tests/specs/benchmarks/verification.k

```diff
@@ -148,6 +148,14 @@ module VERIFICATION-HASKELL [symbolic, kore]
 
     rule maxUInt160 &Int Y => Y requires 0 <=Int Y andBool Y <=Int maxUInt160 [simplification]
 
+    rule ( #sizeByteArray(#ecrec( _, _, _, _ )) <=Int 32 ) => true  [simplification]
+
+    rule MEM [ K1 := BUF1:ByteArray                 ] [ K2 := #ecrec(HASH, SIGV, SIGR, SIGS) ] 
+      => MEM [ K2 := #ecrec(HASH, SIGV, SIGR, SIGS) ] [ K1 := BUF1                           ] 
+        requires (32 <=Int K1 -Int K2) andBool (K2 <Int K1) [simplification]
```

Comment:
Isn't `K2 <Int K1` redundant with `32 <=Int K1 -Int K2` here?

Reply:
Yes, it can be removed.

Reply:
Unresolving this until the code shows up.

Reply:
Should be there now.